js/sales.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+var downtown = {
+  hrs: ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'],
+  minCust: 23,
+  maxCust: 65,
+  avgSalesPerCust: 6.3,
+  avgCustVolume: function(maxCust, minCust) {
+    return Math.ceil(Math.random() * (this.maxCust - this.minCust) + this.minCust);
+  },
+  
+  soldCookiesPerHour: function() {
+    return this.avgCustVolume() * this.avgSalesPerCust;
+  },
+
+  hourlyData: function() {
+    var hourlyTotals = [];
+    var totalCookies = 0;
+    for (var i = 0; i < this.hrs.length; i++) {
+      hourlyTotals.push(Math.round(this.soldCookiesPerHour()));
+      totalCookies += hourlyTotals[i];
+    }
+    return [hourlyTotals, totalCookies];
+  },
+
+  render: function() {
+    var ulEl = document.getElementById('downtown');
+    for (var i = 0; i < this.hourlyData()[0].length; i++) {
+      var liEl = document.createElement('li');
+      liEl.textContent = this.hrs[i] + ': ' + this.hourlyData()[0][i] + ' cookies';
+      ulEl.appendChild(liEl);
+    }
+
+    var totalLiEl = document.createElement('li');
+    totalLiEl.textContent = 'Total: ' + this.hourlyData()[1] + ' cookies';
```

Comment:
Since you are calling `this.hourlyData()`twice, the random numbers for the cookies are getting generated twice. Thus, the total that you are displaying is not actually the sum of the numbers being displayed above.

Reply:
I noticed that during lab today; it should be working correctly now.